src/Compilers/Core/Portable/InternalUtilities/SpanUtilities.cs

```diff
@@ -20,5 +20,8 @@ internal static class SpanUtilities
 
             return true;
         }
+
+        public static bool All<TElement>(this ReadOnlySpan<TElement> span, Func<TElement, bool> predicate)
+            => span.All(predicate, static (item, predicate) => predicate(item));
```

Comment:
I assume this isn't involved in any particuarly hot paths? Otherwiwse consider just duplicating the method to avoid the double function call.